src/lib/ui/ClimateSettingForm/ClimateSettingScheduleFormDefaultClimateSetting.tsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from 'react'
+import type { HvacModeSetting } from 'seamapi'
+
+import { useDevice } from 'lib/seam/devices/use-device.js'
+import { Button } from 'lib/ui/Button.js'
+import { InputLabel } from 'lib/ui/InputLabel.js'
+import { ContentHeader } from 'lib/ui/layout/ContentHeader.js'
+import { ClimateSettingControlGroup } from 'lib/ui/thermostat/ClimateSettingControlGroup.js'
+
+interface ClimateSettingScheduleFormDefaultClimateSettingProps {
+  title: string
+  deviceId: string
+  onBack: () => void
+  onCancel: (() => void) | undefined
+  onSave: () => void
+}
+
+export function ClimateSettingScheduleFormDefaultClimateSetting({
+  title,
+  deviceId,
+  onBack,
+  onCancel,
+  onSave,
+}: ClimateSettingScheduleFormDefaultClimateSettingProps): JSX.Element {
+  const { device } = useDevice({
+    device_id: deviceId,
+  })
+
+  const [mode, setMode] = useState<HvacModeSetting>('heat_cool')
+  const [heatValue, setHeatValue] = useState(70)
+  const [coolValue, setCoolValue] = useState(75)
```

Comment:
I don't really consider this part of the larger form, since it may oftentimes be skipped, so I'm just using state variables here. I think it's fine either way honestly.